account-kit/smart-contracts/src/ma-v2/account/nativeSMASigner.ts

```diff
@@ -26,7 +26,7 @@ import {
  *
  * const MNEMONIC = "...":
  *
- * const account = createSMAV2Account({ config });
+ * const account = createSModularAccountV2({ config });
```

Comment:
❗❗❗ is this a typo? i see `createSModularAccountV2` in 4 examples, but i don't actually see that function anywhere?
```suggestion
 * const account = createModularAccountV2({ config });
```

Reply:
good catch